draft-ietf-mls-protocol.md

```diff
@@ -1543,13 +1545,17 @@ The following sections describe the encryption and signing processes in detail.
 
 ## Content Signing
 
-The `signature` field in an MLSPlaintext object is computed using the
-signing private key corresponding to the credential at the leaf in
-the tree indicated by the sender field.  The signature covers the
-plaintext metadata and message content, which is all of
-MLSPlaintext except for the `signature` field.  The signature also covers the
-GroupContext for the current epoch, so that signatures are specific to a given
-group and epoch.
+The `signature` field in an MLSPlaintext object authenticates the sender of the
+MLSPlaintext message.  The input to the signature is an MLSPlaintextTBS
+structure, which encodes the content and metadata of the message as well a
+context for the signature that associates to the group messages sent within the
+group.
+
+In cases where the sender is a member of the group, the context contains the
+GroupContext for the current epoch and a "membership token" that authenticates
+that the sender is a member of the group.  As a result, for messages sent within
+a group, the signature authenticates not only the member's individual identity,
+but also their membership in the group.
```

Comment:
Actually this isn't quite right because the signature doesn't cover the membership token